tests/pyfunc/test_model_export_with_class_and_artifacts.py

```diff
@@ -1286,7 +1286,7 @@ def test_functional_python_model_list_to_list_pep585(tmp_path):
     assert loaded_model.predict([{"x": "a"}, {"x": "b"}]) == ["a", "b"]
 
 
-def list_dict_to_list(x: List[Dict[str, str]]) -> List[str]:
+def list_dict_to_list(x: List[Dict[str, str]]) -> List[str]:  # UP006
     return ["".join((*d.keys(), *d.values())) for d in x]  # join keys and values
```

Comment:
The same motivation as https://github.com/mlflow/mlflow/pull/13600/files#r1821747482.